code/game/objects/items/oddities.dm

```diff
@@ -119,8 +119,8 @@
 
 /obj/item/weapon/oddity/common/paper_omega
 	name = "collection of obscure reports"
-	desc = "Even the authors seem to be rather skeptical about their findings. Yet they are not connected to each other, but results are similar."
-	icon_state = "paper_omega"
```

Comment:
Find the icon state that it's supposed to be and set it to that

Reply:
sure, I'll dig. However, I dont know what the intended icon is. Obciously something named "paper omega" but it appears as a blank icon in game with no clickable icon outside of storage containers and the right-click menu